applications/matter_weather_station/src/weather-station.matter

```diff
@@ -1153,7 +1158,7 @@ endpoint 0 {
     callback attribute acceptedCommandList;
     callback attribute attributeList;
     ram      attribute featureMap;
-    ram      attribute clusterRevision default = 1;
+    ram      attribute clusterRevision default = 3;
```

Comment:
I don't think it's right. The spec defines just a single revision for this cluster.